projects/core/src/lib/backend/operator.model.ts

```diff
@@ -315,8 +315,8 @@ export type InputResolutionDict = {type: 'source'} | {type: 'value'; x: number;
 export interface RasterUnScalingDict extends OperatorDict {
     type: 'RasterScaling';
     params: {
-        slope: RasterMetadataKey | {type: 'constant'; value: number};
-        offset: RasterMetadataKey | {type: 'constant'; value: number};
+        slope: RasterMetadataKey | {type: 'constant'; value: number} | undefined;
```

Comment:
maybe use the ?-notation instead? `slope?: ...`

Reply:
[0305d3c](https://github.com/geo-engine/geoengine-ui/pull/584/commits/0305d3ccc12fb907ee7fdcfc3d04307180cc2768)